stately-collections/build.gradle

```diff
@@ -56,7 +61,7 @@ kotlin {
             dependencies {
         		implementation 'org.jetbrains.kotlin:kotlin-test-common'
         		implementation 'org.jetbrains.kotlin:kotlin-test-annotations-common'
-                implementation 'co.touchlab:testhelp:0.2.8-SNAPSHOT'
+                implementation 'co.touchlab:testhelp:0.2.8'
```

Comment:
I think this was a left over. At least the other module uses the non snapshot version.